zcash_primitives/src/jubjub/fs.rs

```diff
@@ -321,8 +321,8 @@ impl Field for Fs {
         loop {
             let mut tmp = {
```

Comment:
This could be replaced with
```
// Mask away the unused most-significant bits.
let tmp = Fs(FsRepr([
    rng.next_u64(),
    rng.next_u64(),
    rng.next_u64(),
    rng.next_u64() & (0xffffffffffffffff >> REPR_SHAVE_BITS),
]));
```
And then we wouldn't need the loop below, but that may be beyond the scope of this PR.